fprettify/__init__.py

```diff
@@ -1455,18 +1568,20 @@ def str2bool(str):
                                                              " | 2: operators, print/read, plus/minus"
                                                              " | 3: operators, print/read, plus/minus, muliply/divide"
                                                              " | 4: operators, print/read, plus/minus, muliply/divide, type component selector")
-    parser.add_argument("--whitespace-comma", type=str2bool, nargs="?", default="None", const=True,
-            help="boolean, en-/disable whitespace for comma/semicolons")
-    parser.add_argument("--whitespace-assignment", type=str2bool, nargs="?", default="None", const=True,
+    parser.add_argument("--whitespace-comma", type=str2mode, nargs="?", default="None", const=True,
+            help="True/False/0or1/asis/r_0or1/r_asis, en-/disable whitespace for comma/semicolons")
```

Comment:
this looks a bit cryptic to me, can you explain what these extended options mean?